extensions/amp-user-notification/0.1/amp-user-notification.js

```diff
@@ -209,20 +209,36 @@ export class AmpUserNotification extends AMP.BaseElement {
    * @return {!Promise}
    */
   postDismissEnpoint_() {
+    const enctype = this.element.getAttribute('enctype') || 'application/json';
     return Services.xhrFor(this.win).fetchJson(
         dev().assertString(this.dismissHref_),
-        {
-          method: 'POST',
-          credentials: 'include',
-          requireAmpResponseSourceOrigin: false,
-          body: /** @type {!JsonObject} */({
-            'elementId': this.elementId_,
-            'ampUserId': this.ampUserId_,
-          }),
-          headers: {
-            'Content-Type': 'application/json;charset=utf-8',
-          },
-        });
+        this.buildPostDismissRequest_(enctype, {
+          elementId: this.elementId_,
+          ampUserId: this.ampUserId_,
+        }));
+  }
+
+  buildPostDismissRequest_(enctype, {elementId, ampUserId}) {
+    const formDataEncType = 'application/x-www-form-urlencoded';
```

Comment:
Move const above in the module. Or just use the literal inline.

Reply:
done